### ISSUE TITLE
helixultimate duplicates the functionalities provided by Joomla’s version of Bootstrap javascript

### ISSUE THREAD

**Issue by jgkiefer** [JoomShaper/helix-ultimate#258]
### Steps to reproduce the issue

The duplicated bootstrap javascript creates conflicts with front-end drop-down menus (article edit, etc) not displaying.

### Expected result
Only one call to bootstrap javascript instead of two. Front-end template drop-down menus work.

### Actual result

Front end menus (article edit, print, etc) not clickable and will not open

### System information (as much as possible)
Joomla 4.1.0
PhP 7.4.27
Helix 2.0.6
SP Page Builder Pro 3.8.3
Template shaper_helixultimate - Default

### Additional comments

This has been verified on Joomshaper.com support form and I was asked to post here on Github

[Here is a link to the support post](https://www.joomshaper.com/forum/question/11875#qa-answer-57887)

**Comment by chattigre** [comment]
Hello,

This bug is creating conflicts with main functionalities from Joomla Core (media fields, modals, accordions...) ! When will it be fixed please ?

Regards.

**Comment by joomleb** [comment]
@mi-prakash 
Why was this report closed? As far as I know it has not been fixed:

PHP 7.4.33 + Joomla 4.2.6 + Helix Ultimate 2.0.11
I added two Custom Modules to add two buttons (at the top right of my site)
Clicking on them they are opening the standard [Bootstrap 5 Dropdowns](https://getbootstrap.com/docs/5.1/components/dropdowns/)

Code example:
<div class="dropdown">
<a type="button" class="fas fa-globe-americas" data-bs-toggle="dropdown" aria-expanded="false"></a>
<div class="dropdown-menu">
{module id="..."}
</div>
</div>

They are working well on the entire site, except on these 2 pages:
- https://develop.egulp.net/turismo/alojamiento  page and all its child
- https://develop.egulp.net/component/solidres/?view=experiences&cat=4  and all its child
where, clicking on them, _nothing happen_.

Many Thanks to Andrew from OrdaSoft [here](https://ordasoft.com/Forum/Joomla-Translate-module/47248-Text-list-with-flag-BUGs.html#47273) I understood it could be a HelixUltimate Bootstrap 5 conflict.

Looking for a solution 
I found the @jgkiefer [suggestion here](https://www.joomshaper.com/supports/forums/question/11875) (Thank You. PS now is line 359) that work on the exclusion of the /home/.../templates/shaper_helixultimate/js/**bootstrap.bundle.min.js** file.
But applying it, the problem is reversed: the two buttons open the dropdowns only on the two pages:
- https://develop.egulp.net/turismo/alojamiento  page and all its child
- https://develop.egulp.net/component/solidres/?view=experiences&cat=4  and all its child
and they _stop working_ on everything else on the site.

Please, Can you suggest a solution here ? Do you have a trick / tip, a quick workaround ?


**PS** - By using Helix Ultimate on top of Joomla 4.x I don't see the need to force overriding of what is _**standard**_ in Joomla 4.
There should be a setting to exclude loading, as was done for "Advanced > Font Settings > Enable Font Awesome". Do you agree ?

**Comment by mi-prakash** [comment]
In the next update hopefully this issue will be resolved
thank you for your patience

**Comment by joomleb** [comment]
@mi-prakash 
Thank you, staying tuned.
Please, let me know if you need full access to my test site (in case give me instruction on where to send details to you)

**Comment by jgkiefer** [comment]
This conflict also affects 3rd party extensions. I have had to comment it out each time I performed an extension upgrade. I first reported this a year ago, almost, now. I had given up hope that this bug would ever get fixed.
Thank you for finally addressing it. I greatly look forward to the release of the fix upgrade.

From above:
"PS - By using Helix Ultimate on top of Joomla 4.x I don't see the need to force overriding of what is standard in Joomla 4.
There should be a setting to exclude loading, as was done for "Advanced > Font Settings > Enable Font Awesome". Do you agree?"

Yes, I agree completely.

**Comment by mi-prakash** [comment]
> This conflict also affects 3rd party extensions. I have had to comment it out each time I performed an extension upgrade. I first reported this a year ago, almost, now. I had given up hope that this bug would ever get fixed. Thank you for finally addressing it. I greatly look forward to the release of the fix upgrade.
> 
> From above: "PS - By using Helix Ultimate on top of Joomla 4.x I don't see the need to force overriding of what is standard in Joomla 4. There should be a setting to exclude loading, as was done for "Advanced > Font Settings > Enable Font Awesome". Do you agree?"
> 
> Yes, I agree completely.

About the settings I do have to talk with the team and give a confirmation. right now we tested with bootstrap conflicts, like bootstrap dropdown, popover, tooltip etc

**Comment by joomleb** [comment]
Hi guys, @mi-prakash 
today [url=https://www.joomshaper.com/blog/helix-ultimate-v2-0-12]Helix Ultimate 2.0.12[/url] has been released. I Tested it and the bootstrap dropdown conflict Bug seems has been fixed with success... Thank You.

Anyway, as far as I understand, this topic remain open waiting news about the setting to exclude Bootstrap loading in Joomla 4

**Comment by Savama51** [comment]
Hi all,

I think that removing the bootstrap could be a long long work, because Helix Ultimate has been initialy developed for Jommla 3 (which didn't included modern boostrap).

Then, HU has been adapted to work with Joomla 4, mixing boostraps, instead of using only J4 included bootstrap...

For now, I believe that the dev team will patch conflicts instead of adding option to exclude boostrap from their codes, because of too much work for a free working extension...

Am I right, JoomShaper dev's team ?

**Comment by mi-prakash** [comment]
@Savama51 
yes, we will work with the bootstrap issue when J3 is finally dead.

**Comment by mi-prakash** [comment]
@joomleb 
We suggest you to close this issue and for the settings feature you are requesting please create a new issue describing that. In that case we can manage the git hub issues more effectively.
Thank you

**Comment by joomleb** [comment]
@mi-prakash 
done #313 
Thank You

**Comment by joomleb** [comment]
@mi-prakash 
as far as I know, this should be confirmed as solved with Helix Ultimate 2.1.1, Can you ?
...and the ticket can be closed, thank you
